gto/git_utils.py

```diff
@@ -1,166 +1,84 @@
 import inspect
 import logging
 from contextlib import contextmanager
-from functools import wraps
 from tempfile import TemporaryDirectory
-from typing import Callable, Dict, List, Tuple
+from typing import Callable, Dict, List, Tuple, Union
 
 import git
+from git import Repo
 
-from gto.commit_message_generator import generate_empty_commit_message
+from gto.config import RegistryConfig
 from gto.constants import remote_git_repo_regex
 from gto.exceptions import GTOException, WrongArgs
 
 
-def clone_on_remote_repo(f: Callable):
-    @wraps(f)
-    def wrapped_f(*args, **kwargs):
-        kwargs = _turn_args_into_kwargs(f, args, kwargs)
+class GitRepoMixin:
+    @classmethod
+    def _from_repo(cls, repo: Union[str, Repo], config: RegistryConfig = None):
```

Comment:
tiny suggestion here: what about renaming it `from_local_repo`?

Reply:
Sure!